resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -5,11 +5,12 @@ pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
 
 restartfile=<%= EZBake::Config[:restart_file] %>
 start_timeout=<%= EZBake::Config[:start_timeout] %>
-stop_timeout=60
 dir=$(dirname "$restartfile")
 realname="<%= EZBake::Config[:real_name] %>"
 PIDFILE="/var/run/puppetlabs/${realname}/${realname}.pid"
 
+[ -e "${INSTALL_DIR}/ezbake-functions.sh" ] && . "${INSTALL_DIR}/ezbake-functions.sh"
```

Comment:
Actually, I think I'd need a `set -e` to have it fail if the script doesn't exist.  I had gone with the `set +e` at the top - which is probably redundant with the default behavior - because some of the code in `kill_pid` relies upon handling non-zero exit codes, where we don't want the script to fail, but was trying to avoid blocking `set -e` / `set +e` calls throughout, like @haus had suggested on another recent PR.

So, yeah, as it stands I guess the `-e` check for the file's existence is a no-op.  I could change it to error out with a specific error message if that makes more sense or just assume the script is there and source it - yolo.  Wasn't sure how much error handling for this was useful to include here.  Suppose the one error message wouldn't be too onerous to look at?